conda-recipe/meta.yaml

```diff
@@ -34,11 +34,11 @@ requirements:
   host:
     - category_encoders >=2.3.0
     - dataclasses >=0.6   # [ py < 37 ]
-    - ipykernel >=4.10.1
+    - ipykernel >=4.10.1   # [ py < 38 ]
     - ipykernel >=5.3.0   # [ py >= 38 ]
-    - ipython >=5.5.0
-    - ipython >=7.15.0   # [ py >= 38 ]
-    - ipywidgets >=7.5.0
+    - ipython >=5.5.0, <8   # [ py < 38 ]
```

Comment:
what is the little 🍦  sign means?

Reply:
less than 8, ipython release 8 and it is changes alot of the api